content/en/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -52,6 +52,17 @@ This means the name must:
 - start with an alphanumeric character
 - end with an alphanumeric character
 
+### RFC 1035 Label Names
+
+Some resource types require their names to follow the DNS
```

Comment:
nit: Since the information/rules are the same could you keep the original heading and list the two RFCs within that section?

Reply:
The rules are different and the difference lies in whether the first character can be a number.

Reply:
OK, but the descriptions under each heading appear to be the same.

Reply:
👍